src/ucs/vfs/fuse/vfs_fuse.c

```diff
@@ -128,6 +126,21 @@ static int ucs_vfs_fuse_read(const char *path, char *buf, size_t size,
     return nread;
 }
 
+static int ucs_vfs_fuse_readlink(const char *path, char *buf, size_t size)
+{
+    ucs_string_buffer_t strb;
+
+    ucs_string_buffer_init(&strb);
```

Comment:
we can initialize a fixed-length string buffer from (buf, size) to avoid extra copy by ucs_snprintf_safe